### ISSUE TITLE
Warning occurred while load the model

### ISSUE THREAD

**Issue by Panepo** [zaidalyafeai/zaidalyafeai.github.io#10]
Hi @zaidalyafeai 

I was trying to inference your model by official tensorflow-js library version 0.13.0.
But warning occurred while load some of the models, something like
`The shape of the input tensor ([null,128,128,32]) does not match the expectation of layer conv2d_2: [null,256,256,3]`

Is there something wrong on my code?
Thanks

**Comment by zaidalyafeai** [comment]
Which model? If it is pix2pix then I used a modified version of tfjs because the official version is not implementing a certain functionality that I needed.

**Comment by Panepo** [comment]
Right

Both the `pix2pix` and `Webcam Style Transfer` have the same warning.

**Comment by JeremyJamesG** [comment]
hey @zaidalyafeai , what modified version of tfjs did you use with pix2pix?

**Comment by zaidalyafeai** [comment]
@JeremyJamesG, I had to edit the source code of tfjs to modify the batchnorm which has a bug. Not sure if it was fixed yet.

**Comment by JeremyJamesG** [comment]
great, thanks @zaidalyafeai

**Comment by JeremyJamesG** [comment]
Hey @zaidalyafeai does that mean the 'Fast Pix2Pix in browser' example you wrote on Medium will not work with the out of box tensorflow js?

**Comment by clkruse** [comment]
@zaidalyafeai would you be able to share more info about your batch norm patch? I'm trying to upgrade to the latest TFJS, and have fixed most of the issues, but my output predictions are unchanging based on the input, and are random noise. 

I also have the shape issue (which led me to this issue), but that also happens without the tfjs script in the repo and the outputs are still valid.

**Comment by zaidalyafeai** [comment]
@clkruse there is a related issue https://github.com/tensorflow/tfjs/issues/562
